doc/man3/X509_LOOKUP_meth_new.pod

```diff
@@ -150,17 +150,17 @@ the X509_LOOKUP context, the type of the X509_OBJECT being requested, parameters
 related to the lookup, and an X509_OBJECT that will receive the requested
 object.
 
-Implementations must add objects it finds to the B<X509_STORE> object
-using X509_STORE_add_cert() or X509_STORE_add_crl().  This "duplicates"
-the object by incrementing its reference counter.  However, the
-X509_STORE_CTX_get_by_subject() function also increases the reference
-count which leads to one too many references being held.  Therefore
-applications should additionally call X509_free() or X509_CRL_free()
-to decrement the reference count again.
+Implementations must add objects they find to the B<X509_STORE> object
+using X509_STORE_add_cert() or X509_STORE_add_crl().  This increments
+its reference counter.  However, the X509_STORE_CTX_get_by_subject()
+function also increases the reference count which leads to one too
+many references being held.  Therefore applications should
+additionally call X509_free() or X509_CRL_free() to decrement the
+reference count again.
```

Comment:
I actually like the term "reference count" much more than "reference counter" and would prefer it used throughout, because the former is a concept of the API contract, whereas the latter is an implementation detail. Would you mind changing that en-passant?

Reply:
Done.  Consistency is good

Reply:
> Done. Consistency is good

Did your really push that? I still see two locations:

```make
~/src/openssl$ grep -nH 'counter' doc/man3/X509_LOOKUP_meth_new.pod 
doc/man3/X509_LOOKUP_meth_new.pod:155:its reference counter.  However, the X509_STORE_CTX_get_by_subject()
doc/man3/X509_LOOKUP_meth_new.pod:163:increments the result's reference counter.
```

Reply:
I forgot to push...  done now